frontend/e2e/tests/knowledge/dingtalk-import.spec.ts

```
@@ -0,0 +1,359 @@
+      bodyError = error
+    } finally {
+      let cleanupError: unknown = null
+      try {
+        await cleanupExternalImportScenario(request, context)
+      } catch (error) {
+        cleanupError = error
+      }
+      if (cleanupError && bodyError) {
+        console.error('Scenario cleanup failed (body failure reported first):', cleanupError)
+      }
+      if (bodyError) throw bodyError
```

Biome (2.5.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)